### PR TITLE
Minor correction to qubit spectroscopy ef and drag calibration

### PR DESCRIPTION
**Pull request by andrea-pasquale**
For qubit spectroscopy ef the correction is just a minus sign.
For drag, given that in the literature the parameter reported is usually not beta but beta/anharmonicity.
I've decided to rescale the plot accordingly, in the platform update I made sure to use the correct beta.
Here is an example where I put on purpose a wrong value for beta: http://login.qrccluster.com:9000/vErrCeyfS1KihyZYqviaig==/

Checklist:
- [x] Reviewers confirm new code works as expected.
- [x] Tests are passing.
- [x] Coverage does not decrease.
- [x] Documentation is updated.
- [x] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [x] Qibo: `master`
    - [x] Qibolab: `main`
    - [x] Qibolab_platforms_qrc: `main`

### CODE REVIEW

File: src/qibocal/protocols/drag.py [hunk 1]
```diff
@@ -41,6 +48,8 @@ class DragTuningResults(Results):
 
     betas: dict[QubitId, float]
     """Optimal beta paramter for each qubit."""
+    betas_not_normalized: dict[QubitId, float]
+    """Betas / qubit anharmonicity."""
```
Comment (alecandido) [inline src/qibocal/protocols/drag.py:52]: Do you really need both?

The anharmonicity should be already known by the platform, isn't it?

File: src/qibocal/protocols/drag.py [hunk 2]
```diff
@@ -69,24 +80,29 @@ def _acquisition(
...
+    data = DragTuningData(
+        anharmonicity={
+            qubit: platform.qubits[qubit].anharmonicity * HZ_TO_GHZ for qubit in targets
+        }
+    )
     # define the parameter to sweep and its range:
     # qubit drive DRAG pulse beta parameter
-    beta_param_range = np.arange(
-        params.beta_start, params.beta_end, params.beta_step
-    ).round(4)
```
Comment (alecandido) [inline src/qibocal/protocols/drag.py:76]: Why did you need to round?

**Review by alecandido — COMMENTED** [review]
Mostly fine, I have a single doubt about results duplication, but otherwise should be good to be merged.

File: src/qibocal/protocols/drag.py (hunk 2 above)
Comment (andrea-pasquale) [reply src/qibocal/protocols/drag.py:76]: I believe it is some weird old remnant of possibly some qblox related issues, which I don't know if they are still there...

File: src/qibocal/protocols/drag.py (hunk 1 above)
Comment (andrea-pasquale) [reply src/qibocal/protocols/drag.py]: Removed it https://github.com/qiboteam/qibocal/pull/927/commits/db0d05279e5d077ba21f1f6efb4d69c0aaa0e3a6

**Review by alecandido — APPROVED** [review]

**Review by Edoardo-Pedicillo — APPROVED** [review]
